src/System.IO.FileSystem.Primitives/src/project.json

```diff
@@ -1,12 +1,14 @@
 {
   "frameworks": {
+    "netcore50": {
+      "dependencies": {
+        "System.Runtime": "4.1.0"
+      }
+    },
     "netstandard1.3": {
       "dependencies": {
-        "System.Runtime": "4.0.20"
-      },
-      "imports": [
-        "dotnet5.4"
-      ]
+        "Microsoft.TargetingPack.Private.CoreCLR": "1.2.0-beta-24603-02"
```

Comment:
>"Microsoft.TargetingPack.Private.CoreCLR": "1.2.0-beta-24603-02" [](start = 8, length = 64)

Why did we make this change to bind to coreclr instead of going through System.Runtime like we did before?